templates/shared/forms/interactive/managed-kafka.html

```diff
@@ -0,0 +1,205 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <hr />
+      <h2 class="p-modal__title" id="modal-title">Canonical’s Managed Kafka</h2>
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <p id="modal-description" class="u-no-max-width u-no-margin--bottom">Apache Kafka, managed by the open source experts.</p>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">What version of Kafka are you running?</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="version-of-kafka-1x">
+            <label for="version-of-kafka-1x">1.x
```

Comment:
Can you stop the new line at the end of the label as it's is appearing in the resulting message.

Reply:
![image](https://user-images.githubusercontent.com/1413534/83038384-cd388b80-a034-11ea-927f-ddbaeb7c75cb.png)

Reply:
done